public/javascripts/Progress/src/Progress.js

```diff
@@ -51,30 +51,57 @@ function Progress (_, $, userRole) {
         });
     }
 
-    function putUserOrg(e) {
+    function putUserTeam(e, newTeam) {
         var parsedId = $(this).attr('id').split("-"); // the id comes in the form of "from-startOrg-to-endOrg"
-        var startOrg = parsedId[1];
-        var endOrg = parsedId[3];
+        var startTeam = parsedId[1];
+        if (newTeam != null) {
+            var endTeam = newTeam;
+        } else {
+            var endTeam = parsedId[3];
+        }
         $.ajax({
             async: true,
-            url: '/userapi/setUserOrg/' + endOrg,
+            url: '/userapi/setUserOrg/' + endTeam,
             type: 'put',
             success: function (result) {
                 window.location.reload();
```

Comment:
I know you didn't write it this way, but let's most `window.location.reload();` to be after the logging. It really feels like the page load should be the last thing to be kicked off!